docker/centos7/Dockerfile

```diff
@@ -128,8 +130,25 @@ RUN source /opt/rh/devtoolset-8/enable && \
         -G Ninja \
         -DLLVM_INCLUDE_EXAMPLES=OFF \
         -DLLVM_INCLUDE_TESTS=OFF \
+        -DLLVM_ENABLE_PROJECTS="clang;clang-tools-extra;lld;lldb" \
+        -DLLVM_STATIC_LINK_CXX_STDLIB=ON \
+        ../llvm && \
+    cmake --build . && \
+    cmake --build . --target install && \
+    cd .. && \
+# It must be a clean directory, otherwise there may be undefined behavior.
+    mkdir build2 && \
+    cd build2 && \
+    cmake \
+        -DCMAKE_BUILD_TYPE=Release \
+        -G Ninja \
+        -DLLVM_INCLUDE_EXAMPLES=OFF \
+        -DLLVM_INCLUDE_TESTS=OFF \
+# build clang13 again to pass component check and avoid potential issues
         -DLLVM_ENABLE_PROJECTS="clang;clang-tools-extra;compiler-rt;libcxx;libcxxabi;libunwind;lld;lldb" \
```

Comment:
Do we need to rebuild everything? Or can we just build the components not being built in the previous step?

Reply:
Remove some components, but clang is required, otherwise component check will fail